inst/WORDLIST

```diff
@@ -0,0 +1,45 @@
+Appsilon
+Boxifying
+ESLint
+Init
+JS
+LatinR
+LibSass
+RStudio
+Renv
+Renviron
+Rprofile
+Stylelint
+UI
+config
+devtools
+entrypoint
+favicon
+formatter
+fullstack
+golem
+io
+isolatable
+js
+linter
+linters
+lockfile
+minified
+modularization
+modularize
+modularized
+namespaced
+npm
+overridable
+preconfigured
+renv
+scalable
+shinyapps
+snippers
```

Comment:
Remove and change to "snippets" in the text :grin:

Reply:
Changed snippers to snippets in vignette.